src/main/java/com/opencart/steps/AdminDashboardBL.java

```diff
@@ -0,0 +1,66 @@
+package com.opencart.steps;
+
+import com.opencart.containers.CurrencyContainerAdmin;
+import com.opencart.datamodel.AdminLoginModel;
+import com.opencart.datamodel.CurrencyModel;
+import com.opencart.pages.AdminDashboard;
+
+public class AdminDashboardBL {
+    private final AdminDashboard adminDashboard;
+
+    public AdminDashboardBL() {
+        adminDashboard = new AdminDashboard();
+    }
+
+    public AdminDashboardBL clickOnSidebarButton (String buttonName) {
+        adminDashboard.getButton(buttonName).click();
+        return this;
+    }
+
+    public void getCurrencyValues() {
+        new CurrencyModel();
```

Comment:
You can return Map<String, String> from that method, and put logic from CurrencyModel class to there instead of creating a separate class <CurrencyModel> for one single method.

Reply:
I also use CurrencyModel class to check the checkbox (before deleting new Currency).  
<img width="786" alt="Screen Shot 2020-11-22 at 12 39 14" src="https://user-images.githubusercontent.com/73054470/99902248-0476b500-2cc5-11eb-8ede-f5e32b19a97d.png">

Reply:
You can add take currency name you want to delete as a method parameter, without using CurrencyModel class.